bin/plot_krun_results

```diff
@@ -698,24 +708,13 @@ class ProcessExecChart(object):
             self._plot_outliers(inset, large_markers=False)
             self._plot_changepoints(inset, large_markers=False)
             pyplot.draw()  # Force xticklabels to be drawn.
-            if self.x_bounds[0] > 0:
```

Comment:
[yay.pdf](https://github.com/softdevteam/warmup_experiment/files/868320/yay.pdf)

Here's some 0.7 data. Do you see any weirdness with the ticks?

I'll plot some more with less regular ticks...